utilities/after_success.sh

```diff
@@ -8,8 +8,31 @@ echo "Travis pull request: " ${TRAVIS_PULL_REQUEST}
 echo "Travis JDK version:  " ${TRAVIS_JDK_VERSION}
 if [ "${TRAVIS_JDK_VERSION}" == "oraclejdk7" -a "${TRAVIS_BRANCH}" == "master" -a "${TRAVIS_PULL_REQUEST}" == "false" ]; then
     mvn cobertura:cobertura coveralls:report
-    mvn site-deploy -DskipTests=true --settings=target/travis/settings.xml
     mvn deploy -DskipTests=true -Dgpg.skip=true --settings target/travis/settings.xml
+
+    # Deploy site if not a SNAPSHOT
+    SITE_VERSION=$(mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -Ev '(^\[|Download\w+:)')
+    if [ "${SITE_VERSION##*-}" != "SNAPSHOT" ]; then
+        mvn site-deploy -DskipTests=true --settings=target/travis/settings.xml
+
+        # Update "latest" webpage
+        git config --global user.name "travis-ci"
+        git config --global user.email "travis@travis-ci.org"
+        git clone --branch gh-pages --single-branch https://github.com/GoogleCloudPlatform/gcloud-java/ tmp_gh-pages
+        cd tmp_gh-pages
+        mkdir -p site/latest/
+        touch site/latest/index.html
+        echo "<html><head><meta http-equiv=\"refresh\" content=\"0; URL='http://GoogleCloudPlatform.github.io/gcloud-java/site/${SITE_VERSION}/index.html'\" /></head><body></body></html>" > site/latest/index.html
+        git add site/latest/index.html
+
+        # Update "Quickstart with Maven" block on landing page to reflect latest version
+        sed -i "s/{{SITE_VERSION}}/$SITE_VERSION/g" site/${SITE_VERSION}/index.html
+        git add site/${SITE_VERSION}/index.html
+
+        git commit -m "Updating to reflect latest website version"
```

Comment:
Sure, I can change this to something like "Update the redirect in latest/index.html and the version in the 'Quickstart with Maven' landing page box to X.X.X."